daemon/events/metrics.go

```diff
@@ -0,0 +1,15 @@
+package events
+
+import "github.com/docker/go-metrics"
+
+var (
+	eventsCounter    metrics.Counter
+	eventSubscribers metrics.Gauge
+)
+
+func init() {
+	ns := metrics.NewNamespace("engine", "events", nil)
+	eventsCounter = ns.NewCounter("events", "The number of events logged")
```

Comment:
Apologies if I'm misunderstanding something, but from reading the code in https://github.com/docker/go-metrics/blob/master/namespace.go, it does seem like it would end up being named `engine_events_events_total` due to the namespace above including the `events` string as a subsystem name already. Prometheus glues together the namespace, subsystem, and metric name given in the counter opts constructed in https://github.com/docker/go-metrics/blob/master/namespace.go#L61-L69 with an underscore.

The namespace definition here seems problematic for the other metrics as well. For example, you'd get `engine_events_engine_cpus`, which should probably just be `engine_cpus` (same for `engine_events_engine_memory`)?